frontend/src/container/NotificationWall.jsx

```diff
@@ -119,73 +115,74 @@ export class NotificationWall extends React.Component {
     const i18nOpts = {
       user: `<span title='${escapedUser}'>${escapedUser}</span>`,
       author: `<span title='${escapedAuthor}'>${escapedAuthor}</span>`,
-      content: `<span title='${escapedContentLabel}' class='contentTitle__highlight'>${escapedContentLabel}</span>`,
+      content: `<span title='${escapedContentLabel}' class=${notification.numberOfContents > 1
+        ? ''
+        : 'contentTitle__highlight'
+      }>${escapedContentLabel}</span>`,
```

Comment:
I'd put quotes for the value of the class attribute (espacially for the case where the value is empty.
I'd also put the "matching" case in the ternary first: 

```suggestion
      content: `<span title='${escapedContentLabel}' class='${notification.numberOfContents <= 1
        ? 'contentTitle__highlight'
        : ''
      }'>${escapedContentLabel}</span>`,
```
or even this? I guess there is no content title if there are no contents.

```suggestion
      content: `<span title='${escapedContentLabel}' class='${notification.numberOfContents === 1
        ? 'contentTitle__highlight'
        : ''
      }'>${escapedContentLabel}</span>`,
```

By the way, should we set content to null or to the empty string if there are no contents?

Reply:
will fix. escapedContentLabel is set to empty string if there are no contents.